src/ui/prompt_view.rs

```diff
@@ -0,0 +1,65 @@
+use super::*;
+use crate::application::App;
+
+pub struct PromptView;
+impl<B: Backend> View<B> for PromptView {
+    fn render(&self, f: &mut Frame<B>, app: &mut App) {
+        let (task_chunk, calendar_chunk, footer_chunk) = DefaultView::layout(f.size());
+        DefaultView::render_main(f, f.size());
+        DefaultView::render_tasks(f, task_chunk, &mut app.state.tasks);
+        DefaultView::render_calendar(f, calendar_chunk);
+        DefaultView::render_footer(f, footer_chunk);
```

Comment:
Instead of repeating DefaultView render method implementation, why not create this object and invoke this method directly

Reply:
done